packages/cloud_functions/cloud_functions/example/lib/main.dart

```diff
@@ -21,6 +21,8 @@ Future<void> main() async {
 
 // ignore: public_member_api_docs
 class MyApp extends StatefulWidget {
+  // ignore: public_member_api_docs
```

Comment:
I'd disable `public_member_api_docs` for the example folder

Reply:
Done